bin/main.rb

```diff
@@ -0,0 +1,92 @@
+# rubocop:disable Metrics/PerceivedComplexity, Metrics/CyclomaticComplexity
+
+# !/usr/bin/env ruby
+
+require '../lib/classes.rb'
```

Comment:
- [ ] This should be require_relative. Otherwise it will return this:

![image](https://user-images.githubusercontent.com/61479648/93762666-698b2d00-fbe6-11ea-940e-6fd1e0bcc2eb.png)